apps/web/src/features/search/components/instant-search-preview.tsx

```
@@ -0,0 +1,163 @@
+import { forwardRef } from 'react';
+import { Link } from '@tanstack/react-router';
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+import { ScrollArea } from '@/components/ui/scroll-area';
+import { Skeleton } from '@/components/ui/skeleton';
+import { VibeResult } from './result-items/vibe-result';
```

ESLint

[error] 'VibeResult' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)